.circleci/test-lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
These test files could be DRY'd up I think by passing in the desired script name as a CLI argument. And then in the CircleCI file it would be something like `./.circleci/run-test-script.sh test-lint`, for example.

Reply:
Good idea, I did this in 32bb1d8639a42841e48bf029c59069add9f842ec